lib/tasks/one_off/fix_incomplete_user_journeys.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :one_off do
+  # https://eaflood.atlassian.net/browse/RUBY-3041
+  desc "Fix user journeys incorrectly marked as incomplete"
+  task fix_incomplete_user_journeys: :environment do
+    incomplete_journeys = WasteExemptionsEngine::Analytics::UserJourney.where(completed_route: nil)
+    incomplete_journeys.each do |journey|
+      # check if journey passed the completion route
+      journey.page_views.each do |page_view|
+        page = page_view.page
+        next unless WasteExemptionsEngine::Analytics::UserJourney::COMPLETION_PAGES.include?(page)
+
+        # complete the journey
+        journey.update(
```

Comment:
Shouldn't this call `journey.complete_journey` instead?

Reply:
complete_journey requires transient_registration as a parameter and cannot be called retrospectively. So, I had to come up with different way to mark journeys as complete